src/videojs-contrib-hls.js

```diff
@@ -58,6 +58,98 @@ Object.defineProperty(Hls, 'GOAL_BUFFER_LENGTH', {
   }
 });
 
+Object.defineProperty(Hls, 'MAX_GOAL_BUFFER_LENGTH', {
```

Comment:
A lot of these are simply copies. We should have a method that generates them, and make all of them one line function calls.